setup.py

```diff
@@ -11,7 +11,7 @@
 
 requirements = [
     # Until COMPAS reaches 1.0, we pin major.minor and allow patch version updates
-    'compas>=0.15.6,<0.16',
+    'compas>=0.16,<0.17',
```

Comment:
```suggestion
    'compas>=0.16.1,<0.17',
```

compas 0.16.1 has just been released